Apps/Admin/Tests/Functional/cypress/integration/e2e/support.cy.js

```diff
@@ -139,4 +140,18 @@ describe("Support", () => {
 
         cy.log("Verify email returns no results test finished.");
     });
+
+    it("Verify dependents query returns results.", () => {
```

Comment:
Have you thought about merging this test into existing 'Verify support query' test?  In that test we test all query types?  Why there are so many assertions is because initially there were issues with admin logout/login, which were causing tests to fail.  Issues like not logging out or session not cleared entirely.